ui/src/app/components/runtime-configuration-panel.rtl.spec.tsx

```diff
@@ -2322,47 +2142,23 @@ describe(RuntimeConfigurationPanel.name, () => {
 
     togglePDRadioButton();
 
-    clickExpectedButton('Next');
-    clickExpectedButton('Update');
-    runtimeApiStub.runtime.status = RuntimeStatus.DELETED;
-    expect(runtimeApiStub.runtime.gceWithPdConfig).not.toBeNull();
-    await waitFor(() => {
-      expect(runtimeApiStub.runtime.status).toEqual(RuntimeStatus.CREATING);
-    });
-    expect(runtimeApiStub.runtime.gceWithPdConfig).toBeUndefined();
-  });
+    await clickExpectedButton('Next');
+    await clickExpectedButton('Update');
 
-  it('should allow skipping disk deletion when detaching', async () => {
```

Comment:
missing equivalent

Reply:
Added this test back in, but skipped the last few steps, because they seemed more concerned with the hook functionality.